controllers/servers/host_definer/storage_manager/host_definer_server.py

```diff
@@ -1,13 +1,47 @@
+from controllers.array_action.errors import HostNotFoundError, HostAlreadyExists
+from controllers.array_action.storage_agent import detect_array_type, get_agent
 from controllers.common.csi_logger import get_stdout_logger
+from controllers.common.node_info import NodeIdInfo
 from controllers.servers.host_definer.types import DefineHostResponse
+from controllers.servers.utils import join_object_prefix_with_name, get_array_connection_info_from_secrets
 
 logger = get_stdout_logger()
 
 
 class HostDefinerServicer:
+    def define_host(self, request):  # pylint: disable=invalid-name
+        node_id_info = NodeIdInfo(request.node_id)
+        initiators = node_id_info.initiators
+        connectivity_type = request.connectivity_type
```

Comment:
we can reduce this variable if it is only used by [`create_host`](https://github.com/IBM/ibm-block-csi-driver/blob/9d536f923daaed6eb29f023a25c19b1922cfff30/controllers/servers/host_definer/storage_manager/host_definer_server.py#L32)

Reply:
done https://github.com/IBM/ibm-block-csi-driver/pull/560